src/Sarif.Viewer.VisualStudio/packages.config

```diff
@@ -8,6 +9,7 @@
   <package id="EnvDTE90a" version="9.0.2" targetFramework="net461" />
   <package id="Microsoft.ApplicationInsights" version="2.5.1" targetFramework="net461" />
   <package id="Microsoft.Build.Framework" version="15.6.82" targetFramework="net461" />
+  <package id="Microsoft.CodeAnalysis.BinSkim" version="1.4.5" targetFramework="net461" />
```

Comment:
> <package id="Microsoft.CodeAnalysis.BinSkim" v [](start = 1, length = 47)

why is this required? didn't we shut down the binskim analysis?

Reply:
It's a dependency of the Sarif.Sdk package. I'll file an issue for this and another nuget package fix we need.